microprofile-health/src/test/java/org/jboss/eap/qe/microprofile/health/HealthUtil.java

```diff
@@ -0,0 +1,34 @@
+package org.jboss.eap.qe.microprofile.health;
+
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.ConfigurationException;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.arquillian.ArquillianContainerProperties;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.arquillian.ArquillianDescriptorWrapper;
+
+public class HealthUtil {
+
+    public static URLProvider urlProvider() {
+        return new URLProvider();
+    }
+
+    public static class URLProvider {
```

Comment:
Is it possible to get rid of nested static class and rather make all methods static so in test this would be used like:
```
HealthUrlProvider.getHealthEndpoint()
```

Reply:
Sure.